### PR TITLE
Put schema editor in an accordion

### PR DESCRIPTION
**Pull request by jurgenwerk**
This PR puts the schema editor in the 3rd column in an accordion so that it is in accordance with our design plans and so that the 3rd column now has affordance and space for other upcoming features in addition to the Schema Editor, such as "Playground, "Catalog Entry", etc. 

Here's how it looks (Playground is included just for demonstration purposes to see how it will look when there are multiple panels - but this PR includes just one - the schema editor):
image

Video:

https://github.com/cardstack/boxel/assets/273660/cf6021ba-2c6a-466a-b749-53983e6bc26a

😬 I have tried hard to use our existing accordion that we have in our Boxel UI collection and change it to fit our needs for this column in code mode, but it turned out to be too challenging for my time box. The behaviour of the accordion that we have in our boxel ui vs the accordion from our designs is too different in fundamental ways. Here are a couple of differences:

1. Height of the accordion is fixed 
2. There can be only one opened item at a time
3. The opened item content must stretch to fill the available space in the space of fixed height
4. It is expected the opened item content will be large and would need a scroller

(All these rules apply to the accordion from designs while they do not apply to the existing accordion)

In summary, I think the accordion from our boxel ui is just fine and follows the behaviour of usual accordions, but its behaviour is too different for what we need here, that's why I decided to make a new specialized implementation of an accordion to fit our specific use case.

### CODE REVIEW

**Review by tintinthong — APPROVED** [review]
👍 noice

**Review by habdelra — APPROVED** [review]
